src/edu/csus/ecs/pc2/core/imports/clics/CLICSJsonUtilities.java

```diff
@@ -0,0 +1,506 @@
+
+        if (groups.length == 0) {
+            // no groups defined, no group winners
+            return;
+        }
+
+        
+        /**
+         * This map contains the group/regional winner.  (The first team to solve a problem in a group/region)
+         */
+        Map<Group, ClientId> groupWinners = new HashMap<Group, ClientId>();
+        
+        try {
+            ContestStandings contestStandings = ScoreboardUtilites.createContestStandings(contest);
+
+            List<TeamStanding> teamStands = contestStandings.getTeamStandings();
+            for (TeamStanding teamStanding : teamStands) {
+                ClientId clientId = createClientId(teamStanding);
+                Group teamGroup = getGroupForTeam(contest, clientId);
+                if (teamGroup != null) {
+                    
+                    /**
+                     * Find group winner (or foundClientId will be null if no winner found yet)
+                     */
+                    ClientId foundClientId = groupWinners.get(teamGroup);
+                    
+                    if (foundClientId == null && isActive(contest,clientId)) {
+                        // found no group winner so if thie team is active it is the group winner
+                        
+                        if (!"0".equals(teamStanding.getSolved())) {
+                            /**
+                             * Team has to solve at least one problem to be the group winner 
+                             */
+                            groupWinners.put(teamGroup, clientId);
+                        }
+                    }
+                }
+            }
+        } catch (JAXBException | IllegalContestState | IOException e) {
+            StaticLog.getLog().log(Level.WARNING, "Error while calculating group winners DSA", e);
+        }
+
+        Set<Group> problemElementIds = groupWinners.keySet();
+        for (Group group : problemElementIds) {
+            // first to solve for group
+            ClientId clientId = groupWinners.get(group);
+            if (clientId != null) {
+                Account account = contest.getAccount(clientId);
+
+//            "citation": "Winner(s) of group University of Luxembourg", 
+//            "id": "group-winner-17"
+
+                String awardId = GROUP_WINNER_TITLE + group.getGroupId();
+                String citation = WINNER_S_OF_GROUP_TITLE + group.getDisplayName();
+
+                CLICSAward groupWinner = new CLICSAward(awardId, citation, "" + clientId.getClientNumber());
+                list.add(groupWinner);
+            }
+        }
+    }
+
+    public static ClientId createClientId(TeamStanding teamStand) {
+        ClientId clientId = null;
+
+        try {
+            return new ClientId(Integer.parseInt(teamStand.getTeamSiteId()), ClientType.Type.TEAM, Integer.parseInt(teamStand.getTeamId()));
+        } catch (Exception e) {
+            ; // ignore excepion return null
```

Comment:
Seems like this `catch` clause should log an error before falling through to return null.  Wouldn't we want to have a record of the fact that either the `teamSiteId` or the `teamId` couldn't be parsed properly?  That would seem to be an (internal) error worth logging...